api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -215,6 +215,19 @@ message ExtAuthz {
   // Whether to increment cluster statistics (e.g. cluster.<cluster_name>.upstream_rq_*) on authorization failure.
   // Defaults to true.
   google.protobuf.BoolValue charge_cluster_response_stats = 20;
+
+  // Whether to encode the raw headers (i.e. unsanitized, not inlined) in authentication request.
```

Comment:
Please document behavior when using an http ext_authz service (not grpc).

Reply:
Ah, this breaks the http client right now. Working on it.